OrbitCore/SamplingProfiler.cpp

```diff
@@ -145,8 +128,8 @@ void SamplingProfiler::ProcessSamples() {
   address_to_module_name_.clear();
 
   // Unique call stacks and per thread data
-  for (const CallstackEvent& callstack : callstack_events_) {
-    if (!HasCallStack(callstack.callstack_hash())) {
+  for (const CallstackEvent& callstack : callstack_data.GetCallstackEvents()) {
+    if (!callstack_data.HasCallStack(callstack.callstack_hash())) {
```

Comment:
This would be a programming error, right? So could you change this into a CHECK here?

Reply:
Yes, replaced with CHECK